workflow/controller/steps.go

```diff
@@ -603,7 +603,7 @@ func (woc *wfOperationCtx) prepareDefaultMetricScope() (map[string]string, map[s
 	localScope[durationCPU] = "0"
 	localScope[durationMem] = "0"
 
-	var realTimeScope = map[string]func() float64{
```

Comment:
let's remove the style change that's unrelated to this PR (maybe the white line changes in functional_test.go as well)
e.g. reduce chance of conflict if we need to cherry pick this change

Reply:
Yes indeed I have the format on save let me clean it up.